src/Globals.h

```diff
@@ -352,6 +352,15 @@ template class SizeChecker<UInt8,  1>;
 	#define assert_test(x) ( !!(x) || (assert(!#x), exit(1), 0))
 #endif
 
+#define UNREACHABLE(x) do { FLOGERROR("Hit unreachable code: {0}, file {1}, line {2}", #x, __FILE__, __LINE__); PrintStackTrace(); std::terminate(); } while (false)
```

Comment:
The current implementation uses `ASSERT` to only kill debug executables, but lets the production executable alive. So why you just call `std::terminate`, which also kills the production executable?

Reply:
Under clang builds it's undefined behaviour so I don't want anyone relying on it working in release mode.

Reply:
Sorry, but I don't understand the logic. It was always UB in clang. But on GCC/MSVC the last code had a backup solution to keep the application running.

Reply:
Sorry, but I don't understand the logic. It was always UB in clang. But on GCC/MSVC the last code had a backup solution to keep the application running.